pkg/provisioner/ironic/ironic_test.go

```diff
@@ -194,11 +194,11 @@ func TestGetVLANs(t *testing.T) {
 	vlans, vid := getVLANs(introspection.BaseInterfaceType{
 		LLDPProcessed: map[string]interface{}{
 			"switch_port_vlans": []map[string]interface{}{
-				map[string]interface{}{
+				{
```

Comment:
Some of the changes in this file look correct, but not related to the original patch. Could you separate those into a different commit as part of this PR, so the history is clear, please?